### ISSUE TITLE
loss.backward()  Process finished with exit code -1073741819 (0xC0000005)

### ISSUE THREAD

**Issue by wydonglove** [eriklindernoren/PyTorch-YOLOv3#362]
when run train.py,  there is a problem in the line:
loss.backward()
where not run sucessful and print the string: Process finished with exit code -1073741819 (0xC0000005)
can anyone help me?

**Comment by wydonglove** [comment]
My environment is python3.7 windows10 torch1.3.0. cuda9.2

**Comment by wydonglove** [comment]
have solved , change the pytorch and cuda version
